server/schema/user/user.datasources.js

```diff
@@ -178,19 +178,31 @@ const updateDetails = (id, fields, session, authToken, mid) =>
     { new: true }
   );
 
-const updateRoles = async (id, roles) => {
+const getCustomClaims = async (id) => {
   try {
     const _user = await UserModel.findById(id, 'email');
     if (!_user) {
       throw APIError('NOT_FOUND', null, { reason: 'The requested user does not exist.' });
     }
```

Comment:
Why retrieve mongodb usermodel? If we're only dealing with roles, firebase is enough. Also, if a check is needed, firebase can be used itself.